src/main/java/com/minecolonies/coremod/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -270,7 +270,7 @@ private void requestEntityToBuildingIfRequired(Template.EntityInfo entityInfo)
                     {
                         stack.setCount(1);
                         request.add(stack);
-                        request.add(new ItemStack(Items.ITEM_FRAME, 1, stack.getItemDamage()));
+                        request.add(new ItemStack(Items.ITEM_FRAME, 1));
```

Comment:
It seems like this should be move outside of the null check. Because even if the frame is empty, we want to request an ItemFrame